justfile

```diff
@@ -1,46 +1,49 @@
 list:
     @just --list
 
+export port := "5555"
```

Comment:
(optional) btw can this be supplied by env var, maybe check Justfile's documentation.